package/dart-sass.sh

```diff
@@ -8,12 +8,15 @@
 # package`.
 
 function follow_links() {
-  file="$1"
+  local file="$1"
+
+  cd $(dirname "$file")
   while [ -h "$file" ]; do
     # On Mac OS, readlink -f doesn't work.
     file="$(readlink "$file")"
+    cd $(dirname "$file")
   done
-  echo "$file"
+  echo "$(pwd -P)/$(basename "$file")"
```

Comment:
> This script isn't deployed on Mac. /usr/local/bin/dart is directly a symlink to the binary.

I don't understand. This script doesn't wrap the `dart` executable, it wraps `dart-sass`. And if this isn't deployed, then how does the user invoke `dart-sass`? There's no single executable to invoke, you have to pass the snapshot to the `dart` executable.

> Concerning this fix, Homebrew uses double relative links redirection. Without the fix, the issue is following:
> ```
> $ dart-sass 
> /usr/local/bin/dart-sass: line 21: <current pwd>/../Cellar/dart-sass/1.0.0-alpha.8/bin/src/dart: No such file or directory
> /usr/local/bin/dart-sass: line 21: exec: <current pwd>/../Cellar/dart-sass/1.0.0-alpha.8/bin/src/dart: cannot execute: No such file or directory
> ```

Why doesn't repeatedly `readlink`ing work? Why does `cd`ing fix it? Doesn't it change the user's working directory in a way they won't want? Why is `local file` required?

I've filed dart-lang/sdk#28747 to track the issue in the Dart SDK.